_studio/mfx_lib/vpp/include/mfx_vpp_main.h

```diff
@@ -116,6 +116,8 @@ class VideoVPPMain : public VideoVPP {
                 nativeSurface->Data.FrameOrder = surface->Data.FrameOrder;
                 nativeSurface->Data.TimeStamp = surface->Data.TimeStamp;
                 nativeSurface->Info = surface->Info;
+                nativeSurface->Data.ExtParam = surface->Data.ExtParam;
+                nativeSurface->Data.NumExtParam = surface->Data.NumExtParam;
```

Comment:
THIS IS A BUG! Please do git blame

Reply:
Fixed (your latest commit is merged here)